app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -41,6 +41,22 @@ def url_options
 
   private
 
+  def analytics
+    Analytics.new(user: identity.user, request: nil, session: {}, sp: nil)
+  end
```

Comment:
I feel like our typical pattern is to log analytics from the controller level. is there a way that we can have this class expose an `analytics_attributes` or something that we can leverage from the controller? then we can attribute the right request, SP, session info as needed as well

Reply:
i found some examples of this pattern, so thought that it would be okay in this scenario, especially since it's a short-term event that will be removed when it's safe to implement the actual fix. 

but i don't feel that strongly about it, and can try to figure out how to expose the needed data at the controller level -- just means it will have to wait until after i'm back!

Reply:
I think some of the context such as SP is valuable, maybe as another option we could popular `sp:` from that data here

Reply:
i agree that we could populate sp data -- i'm not sure what "as another option" means exactly here? do you mean keeping this event in the presenter with the included data?

Reply:
I think the suggestion is to use `identity` and pass in `sp: identity.service_provider` or similar.

Reply:
i know that the Analytics object adds some extra details if the `sp` value is there, so i can add it in, in case any of those details end up being useful!